examples/nnunet_example/README.md

```diff
@@ -38,3 +38,32 @@ python -m examples.nnunet_example.client --dataset_path examples/datasets/nnunet
+
+## Defenitions
+
+### Logits:
+
+The outputs of a model <ins>prior</ins> to the activation function. Values are unconstrained (-inf, inf)
+
+### Probabilities:
+
+Probabilities are values constrained to the range (0, 1) that represent the models confidence of a pixel/voxel being part of a particular class. Similar to other DNN's, they do not necessarily represent actual probabilities, however it is sometimes convenient to interpret them as such. The predicted probabilities are the outputs of a model <ins>after</ins> a normalizing activation function such as softmax or sigmoid. A 2d example is shown below. In some instances one might have labels that are probabilities as opposed to integer class labels in which case we would refer to them as ground-truth probabilities.
+
+### Detection Maps:
+
+Images that contain an arbitrary number of distinct detected volumes generally derived from the predicted probabilities of a segmentation model. Values are constrained to range [0, 1]. Detected volumes are defined as:
+- Each detected volume is a connected component that must be non-connected and non-overlapping (mutually exclusive) with other volumes of the same class. (Therefore detection maps for multiclass segmentation must be one hot encoded)
+- Each pixel/voxel within a volume must have the same predicted probability. Therefore there is a single confidence/likelihood score for each volume.
+
+Detected volumes typically also have a minimum size determined by the number fo pixels/voxels that are a part of the volume. Detection maps may be computed from probabilities in a variety of ways. One example used for 3d medical images can be found in the [report guided annotation](https://github.com/DIAGNijmegen/Report-Guided-Annotation) API. An example of a 2d detection map is shown below.
+
+### Segmentations:
+
+Images in which pixels have been labelled or assigned one or more specific integer classes. If one hot encoded they must be binary {0, 1} or boolean {False, True} tensors. If not one hot encoded they must be be tensors containing only integers that represent the class labels (eg. constrained to {0, 1, 2, ..., N}). The labels/targets for segmentation models may be referred to as ground-truth segmentations. Predicted segmentations refers to model outputs (which are usually probability maps) that have been processed or thresholded in some way to adhere to the definition of a segmentation. They usually represents the model's final prediction of the class with no information on confidence. An example of a binary or one-hot-encoded predicted segmentation is shown below.
+
+#### Examples of different output types in the case of 2d binary segmentation:
+
+
+<img src="assets/probs.png" width="250"/> | <img src="assets/detmap.png" width="250"/>| <img src="assets/seg.png" width="210"/>|
```

Comment:
These pictures are not rendering, the assets folder does not exist in this directory (or anywhere as far as I can tell). I noticed that is also the case in the main README for PICAI in the [Data section](https://github.com/VectorInstitute/FL4Health/tree/main/research/picai#data)

Reply:
Lol i'm just now realising *.png is in the gitignore